ui/DesignSystem/Component/List.svelte

```diff
@@ -1,11 +1,12 @@
-<script>
+<script lang="ts">
   import { createEventDispatcher } from "svelte";
 
   const dispatch = createEventDispatcher();
 
-  export let dataCy = null;
-  export let items = null;
-  export let style = null;
+  export let items: any[];
```

Comment:
Is this not of type `SvelteComponent | HTMLElement`?

Reply:
In this context, `items` are passed as props to the `slot` as an array of `Project`s. I think `any` is fine here because it's a flexible component that we use for several different use cases.